### DISCUSSION TITLE
Get result of word segmentation

### DISCUSSION THREAD

**Discussion by miaotiao** [meilisearch/product#482] [Feedback & Feature Proposal]
How can i get result of word segmentation.

**Comment by gmourier** [comment]
Hi @miaotiao 👋

This part happens in the tokenizer but is not exposed as a result. Why would you need it? If it's a misunderstanding about word segmentation, please correct me!

Thanks!

**Comment by miaotiao** [comment]
Hello @gmourier 

Sorry for my poor English.
Sometimes search a sentence not a word and i want to know what word is searched

Thanks! 😁

**Reply by gmourier** [reply]
Hello @miaotiao

Why would you like to know what word is being searched?

Currently, documents are matched in this way:

For the query: `A Phrase I Search`

Meilisearch will match the documents that contain all the words of the query `A Phrase I Search`.
Then `A Phrase I`
Then `A Phrase`
And finally `A`

The top documents are the ones that will contain the most words and so on. This work is done by the `words` ranking rule.

Are you talking about knowing how a document was ranked by the engine within the search response? If so, why? 

Thanks!
